converter/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -0,0 +1,166 @@
+package gov.cms.qpp.conversion.model.error;
+
+import java.util.Objects;
+
+import com.google.common.base.MoreObjects;
+
+/**
+ * Error codes that may be returned by the converter
+ */
+public enum ErrorCode implements LocalizedError {
+
+	ENCODER_MISSING("Failed to find an encoder"),
+	NOT_VALID_XML_DOCUMENT("The file is not a valid XML document"),
+	UNEXPECTED_ERROR("Unexpected exception occurred during conversion"),
+	UNEXPECTED_ENCODE_ERROR("Unexpected exception occured during encoding"),
+	NOT_VALID_QRDA_DOCUMENT("The file is not a QRDA-III XML document"),
+	MEASURE_GUID_MISSING("The measure reference results must have a measure GUID"),
+	CHILD_MEASURE_MISSING("The measure reference results must have at least one measure"),
+	AGGREGATE_COUNT_VALUE_NOT_SINGULAR("A single aggregate count value is required"),
+	AGGREGATE_COUNT_VALUE_NOT_INTEGER("Aggregate count value must be an integer"),
+	ACI_MEASURE_PERFORMED_RNR_MEASURE_PERFORMED_MISSING("ACI Measure Performed RnR's Measure Performed is required"),
+	ACI_MEASURE_PERFORMED_RNR_MEASURE_PERFORMED_REPEATED("ACI Measure Performed RnR's Measure Performed can only be present once"),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 128 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)